openlibrary/core/processors/readableurls.py

```diff
@@ -161,9 +165,13 @@ def match(path):
     if thing:
         prefix = thing.key
 
-    if thing and thing.type.key == type:
-        title = thing.get(property) or default_title
-        middle = '/' + h.urlsafe(title.strip())
+    if thing and thing.type.key == _type:
+        title = thing.get(_property) or default_title
+        try:
+            from six.moves.urllib_parse import quote_plus
```

Comment:
On second thought -- this needs to fail on python2 (the solution should be conditional on python3, thus this is basically a py3 check)

```suggestion
           from urllib.parse import quote_plus
```

Reply:
OK.  Now it makes sense.  Perhaps add a comment to explain.